handlers/whatsapp/whatsapp_test.go

```diff
@@ -373,19 +373,19 @@ var defaultSendTestCases = []ChannelSendTestCase{
 	{Label: "Link Sending",
 		Text: "Link Sending https://link.com", URN: "whatsapp:250788123123", Path: "/v1/messages",
 		Status: "W", ExternalID: "157b5e14568e8",
-		ResponseBody: `{ "messages": [{"id": "157b5e14568e8"}] }`, ResponseStatus: 201,
+		ResponseBody: `{ "contacts":[{"input":"250788123123","wa_id":"250788123123"}], "messages": [{"id": "157b5e14568e8"}] }`, ResponseStatus: 201,
 		RequestBody: `{"to":"250788123123","type":"text","preview_url":true,"text":{"body":"Link Sending https://link.com"}}`,
 		SendPrep:    setSendURL},
 	{Label: "Plain Send",
 		Text: "Simple Message", URN: "whatsapp:250788123123", Path: "/v1/messages",
 		Status: "W", ExternalID: "157b5e14568e8",
-		ResponseBody: `{ "messages": [{"id": "157b5e14568e8"}] }`, ResponseStatus: 201,
+		ResponseBody: `{ "contacts":[{"input":"250788123123","wa_id":"250788123123"}], "messages": [{"id": "157b5e14568e8"}] }`, ResponseStatus: 201,
 		RequestBody: `{"to":"250788123123","type":"text","text":{"body":"Simple Message"}}`,
 		SendPrep:    setSendURL},
 	{Label: "Unicode Send",
 		Text: "☺", URN: "whatsapp:250788123123",
 		Status: "W", ExternalID: "157b5e14568e8",
-		ResponseBody: `{ "messages": [{"id": "157b5e14568e8"}] }`, ResponseStatus: 201,
+		ResponseBody: `{ "contacts":[{"input":"250788123123","wa_id":"250788123123"}], "messages": [{"id": "157b5e14568e8"}] }`, ResponseStatus: 201,
```

Comment:
Currently in our logs I see the responses we get are 

```
{"messages":[{"id":"gBGHI0gDaJdFfwIJIN2QOPc_Il5D"}],"meta":{"api_status":"stable","version":"2.37.2"}}
```

So we should not change these existing tests to have the contacts 

```
"contacts":[{"input":"250788123123","wa_id":"250788123123"}],
```

Reply:
I agree, thank you.